thirtybees/contributing/contributing.md

```diff
@@ -0,0 +1,103 @@
+---
+title: thirty bees - How To Contribute Source Code
+tags:
+  - "Installation"
+  - "Getting Started"
+audience: developers
+permalink: /contributing/
+---
+# Contributing to thirty bees
+thirty bees is an open-source e-commerce solution, forked from [PrestaShop 1.6.x](https://github.com/PrestaShop/PrestaShop/tree/1.6.1.x) . We'd like to encourage everyone to participate in the project, including contributing your own improvements to the source code.
+ 
+## Procedures
+ In order to contribute to this project, you need to be familiar with Git, an open source version control system used extensively by thirty bees developers, as well as GitHub:
+ - A nice tutorial about Git can be found here: https://try.github.io/
+ - GitHub help: https://help.github.com/
+ 
+ Contributing to this project via Git can be done as follows:
+ - Fork this project to your own GitHub account
```

Comment:
@firstred the list items looked good in my GUI MD editor. But showing up as a whole paragraph in jekyll. Can you view http://docs.thirtybees.com/contributing/ and the source to see what went wrong with the list item around line 18?